apps/web/package.json

```diff
@@ -7,7 +7,7 @@
     "build": "next build",
     "start": "next start",
     "typecheck": "tsc --noEmit",
-    "lint": "eslint . --ext .js,.ts,.tsx"
+    "lint": "eslint src/ --ext .js,.ts,.tsx"
```

Comment:
@bigint I have an issue when run Eslint not response

Reply:
Hope this is resolved 🤞🏼